binders/kafka-binder/spring-cloud-stream-binder-kafka-core/src/main/java/org/springframework/cloud/stream/binder/kafka/properties/KafkaBinderConfigurationProperties.java

```diff
@@ -665,4 +672,35 @@ public KafkaProducerProperties getExtension() {
 
 	}
 
+	public static class Metrics {
+		/**
+		 * When set to true, the offset lag metric of each consumer topic is computed whenever the metric is accessed (default: true).
+		 * When set to false only the periodically calculated offset lag is used.
+		 * See {@link #scheduledOffsetLagComputationInterval} for more information.
+		 */
+		private boolean realtimeOffsetLagComputationEnabled = true;
```

Comment:
How about something like `defaultOffsetLagMetricsEnabled`? Including the literal `metrics` in the property may clarify what this property is used for. We can also go with `realtimeOffsetLagMetricsEnabled`

Reply:
I like `defaultOffsetLagMetricsEnabled`, adjusting it! Thank you